validation/curvilinear_diffusion/spot_tracer_diffusion.jl

```diff
@@ -1,5 +1,3 @@
-# # Meridional diffusion
-
 using Oceananigans
 using Oceananigans.TurbulenceClosures: Horizontal
 using Oceananigans.Models.HydrostaticFreeSurfaceModels: HydrostaticFreeSurfaceModel, PrescribedVelocityFields
```

Comment:
I think that these two lines can be deleted

Reply:
OK Done!